resolver/resolver.cc

```diff
@@ -3846,10 +3891,11 @@ ast::ParsedFilesOrCancelled Resolver::runIncremental(core::GlobalState &gs, vect
     return result;
 }
 
-ast::ParsedFilesOrCancelled Resolver::runIncrementalWithoutStateMutation(const core::GlobalState &gs,
-                                                                         vector<ast::ParsedFile> trees) {
+// TODO(jez) This is dupliicated with Resolver::runIncremental. Can we make them share more code?
+ast::ParsedFilesOrCancelled Resolver::runIncrementalBestEffort(const core::GlobalState &gs,
+                                                               vector<ast::ParsedFile> trees) {
     auto workers = WorkerPool::create(0, gs.tracer());
-    // trees = ResolveConstantsWalk::resolveConstants(gs, std::move(trees), *workers);
+    trees = ResolveConstantsWalk::resolveConstants(gs, std::move(trees), *workers);
```

Comment:
This was accidentally commented out in the original change. It's quite important that it not be commented out (this is basically all the work that resolver does 😅 )